Makefile

```diff
@@ -13,6 +13,3 @@ frontend:
 #Install dependencies, run eslint, build and test frontend
 frontend.deploy:
 	$(MAKE) frontend
-
```

Comment:
also, why do we have 2 tasks here, one that just calls the other? can't we just get rid of `frontend.deploy` here and always use `frontend`?